src/lib/network_pool/snark_pool.ml

```diff
@@ -561,6 +582,9 @@ end
 
 (* Only show stdout for failed inline tests. *)
 open Inline_test_quiet_logs
+module Mock_snark_pool =
```

Comment:
This is generally a bad idea. Calling a functor also instantiates any 'toplevel code' in that module, and so any modules instantiated explicitly for tests should really be instantiated inside the relevant `test_module` (for `ppx_test`; equivalently in the test program for alcotest).